src/elements/JobsPanelTitle/JobsPanelTitleView.js

```diff
@@ -20,14 +22,15 @@ const JobsPanelTitleView = ({
   setOpenScheduleJob,
   versionOptions
 }) => {
+  const jobPanelClassName = classNames({
+    'job-panel__title-wrapper': 'true',
```

Comment:
I think you mean:
```suggestion
    'job-panel__title-wrapper': true,
```
Although the result is the same, because the string `'true'` is truthy (`Boolean('true') === true`).